<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>115</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>650</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>快速记录</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 85);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyLineEdit" name="x_val">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="y_val"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="h_val"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="d_val"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>481</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="saveFile">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>选择保存的文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pointFile">
         <property name="text">
          <string>选择点位文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>标准点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="point">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>CEP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CEP_res">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="H_res">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="fold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>⇅</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="about">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>About</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="isCalc">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>计算结果</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="adjustData">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>整理导航精度数据</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
   <slots>
    <signal>focus()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
